components/cluster/command/root.go

```diff
@@ -50,12 +50,12 @@ var (
 	skipConfirm bool
 )
 
-var tidbSpec *meta.SpecManager
+var tidbSpec *spec.SpecManager
+var deployer *deploy.Deployer
 
 func scrubClusterName(n string) string {
 	return "cluster_" + telemetry.HashReport(n)
 }
-
```

Comment:
keep the empty line for better formatting... BTW, I think `go fmt` should add the empty line automatically?